### PR TITLE
Fix skipping mouse wheel event of ScrollView inside the contenteditable container (T721099)

### PR DESCRIPTION
**Pull request by jsdmitry**

### CODE REVIEW

**Review by DokaRus — DISMISSED** [review]

File: js/events/utils.js [hunk 1]
```diff
@@ -151,14 +147,16 @@ const hasTouches = (e) => {
...
+    const target = e.target;
+    const $target = $(target);
     const touchInInput = $target.is("input, textarea, select");
 
     if($target.is(".dx-skip-gesture-event *, .dx-skip-gesture-event")) {
         return true;
     }
     if(e.type === 'dxmousewheel') {
-        return $target.is("input[type='number'], textarea, select") && $target.is(':focus');
+        const isContentEditableFocused = target.isContentEditable && $target.closest("div[contenteditable='true']").is(':focus');
```
Comment (San4es) [inline js/events/utils.js:162]: isFocusedContentEditable?

File: js/events/utils.js [hunk 2]
```diff
@@ -151,14 +147,16 @@ const hasTouches = (e) => {
...
+        return $target.is("input[type='number'], textarea, select") && $target.is(':focus') || isContentEditableFocused;
```
Comment (San4es) [inline js/events/utils.js]: return isFocusedInputElement || isFocusedContentEditable

File: js/events/utils.js [hunk 3]
```diff
@@ -113,10 +113,6 @@ const isTouchEvent = (e) => {
     return isNativeTouchEvent(e) || ((isPointerEvent(e) || isDxEvent(e)) && e.pointerType === "touch");
 };
 
-const isKeyboardEvent = (e) => {
```
Comment (San4es) [inline js/events/utils.js]: `DevExpress.events.utils.isKeyboardEvent`

@jsdmitry @DokaRus your thoughts?

**Review by San4es — APPROVED** [review]
LGTM

**Pull request merged**